app/server/src/configApp.ts

```diff
@@ -0,0 +1,66 @@
+import express from "express";
+import cors from "cors";
+import morgan from "morgan";
+import passport from 'passport';
+import session from 'express-session';
+import PassportGoogle from 'passport-google-oauth20';
+import PassportGithub from 'passport-github';
+import MockStrategy from 'passport-mock-strategy';
+
+import config from './resources/config.json';
+
+export const configApp = (app => {
```

Comment:
Maybe rename this method to make it sound more verb-y and less noun-y?

```suggestion
export const configureApp = (app => {
```